common-content/en/module/js3/asynchrony/index.md

```diff
@@ -44,42 +44,3 @@ A single thread can do one thing at a time. JavaScript is a single threaded lang
 {{</tooltip>}}.
 
 When we call a function, the function will run to completion before the next line of code is executed. But what if we need to wait for something to happen? What if we need to wait for our data to arrive before we can show it? In this case, we can use **asynchronous execution**.
-
-### Event Loop
-
-We have already used asynchronous execution. We have defined `eventListener`s that _listen_ for events to happen, _then_ execute a callback function.
-
-```js
-const search = document.getElementById("search");
-search.addEventListener("input", handleInput);
-```
-
-When we called `addEventListener` it didn't immediately call `handleInput`.
-
-But here's a new idea: eventListeners are part of the [Event API](https://developer.mozilla.org/en-US/docs/Web/API/Event). They are not part of the JavaScript language! 🤯 This means you can't use them in a Node REPL. But they are implemented in web browsers. The core of JavaScript (e.g. strings and functions) is the same everywhere, but different contexts may add extra APIs.
-
-When you set an eventListener you are really sending a call to a Web API and asking it do something for you.
-
-```js
-const search = document.getElementById("search");
-search.addEventListener("input", handleInput);
-```
-
-The callback `handleInput` does not run until the user types. With `fetch`, the callback function does not run until the data arrives. In both cases, we are waiting for something to happen before we can run our code.
-
-We use a function as a way of wrapping up the code that needs to be run later on. This means we can tell the browser _what_ to do when we're done waiting.
-
-**👉🏽 [Visualise the Event Loop](http://latentflip.com/loupe/?code=JC5vbignYnV0dG9uJywgJ2NsaWNrJywgZnVuY3Rpb24gb25DbGljaygpIHsKICAgIGNvbnNvbGUubG9nKCdZb3UgY2xpY2tlZCB0aGUgYnV0dG9uIScpOyAgICAKfSk7Cgpjb25zb2xlLmxvZygiSGkhIik7Cgpjb25zb2xlLmxvZygiV2VsY29tZSB0byB0aGUgZXZlbnQgbG9vcCIpOw%3D%3D!!!PGJ1dHRvbj5DbGljayBtZSE8L2J1dHRvbj4%3D)**
-
-### 🧠 Recap our concept map
```

Comment:
You've removed the map -  can I have it back? Not everyone thinks in the same way. We can have multiple ways of modelling ideas.

Reply:
I did leave this visualisation in but I bumped it over to the callback block because it references the event loop and asynchronous execution in more depth. The async block is now called synchronous-execution because it only really talks about synchronous execution before async execution is developed in the next block.